### DISCUSSION TITLE
archery   nginx  forbidden ？

### DISCUSSION THREAD

**Discussion by stan-c** [hhyo/Archery#2656] [Q&A]
Can archery be accessed normally using port 9123, configure nginx to access Forbidden?
```
location / {
proxy_pass http://127.0.0.1:9123;  
proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
}
```

**Comment by LeoQuote** [comment]
Please consider describing your issue with your expectation and actual behavior so we can better  understand it.

**Comment by jackiechan1981** [comment]
When 403 access forbidden appears after archery is deployed in nginx, it is because of CSRF security validation of the Python framework Django.
I have encountered the same problem here, as long as the port of nginx server is also configured as 9123, it can pass CSRF authentication. Just like the configuration below：
<img width="506" alt="image" src="https://github.com/hhyo/Archery/assets/29398311/bf70c9fa-2e1d-46f3-9943-caf70431fae8">
